helix-core/src/main/java/org/apache/helix/tools/ClusterVerifiers/ZkHelixClusterVerifier.java

```diff
@@ -188,6 +194,12 @@ public boolean verifyByPolling() {
     return verifyByPolling(DEFAULT_TIMEOUT, DEFAULT_PERIOD);
   }
 
+  public void close() {
```

Comment:
nit, do we need to change any test cases to call close? Or they are all using the constructor with a HelixZkClient input?

Reply:
I am running mvn test to be sure :)
But this is a new method, so it shouldn't affect existing tests.

Reply:
mvn test passed.